server/src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -3388,7 +3391,7 @@ public void onFailure(Exception e) {
         });
     }
 
-    protected void waitForCommitDurability(long generation, ActionListener<Void> listener) {
+    private void notWaitForCommitDurability(long generation, ActionListener<Void> listener) {
```

Comment:
Can we call this `checkGenerationFlushed` instead?

Reply:
Yep updated in 16c6350f8dcd6dae84e9a4b38d81c613117dfe4d